Subject: Make OSLEC the default DAHDI echo canceller
Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Forwarded: no

Our dahdi-linux packages provide OSLEC, which is generally a better echo
canceller than those included with DAHDI. Thus we should make it the 
default echo canceller in generated configurations.

Debian-specific configuration tweak.

--- a/xpp/perl_modules/Dahdi/Config/Params.pm
+++ b/xpp/perl_modules/Dahdi/Config/Params.pm
@@ -106,7 +106,7 @@ sub item($$) {
 			group_lines		=> '0',
 			brint_overlap		=> 'no',
 			bri_sig_style		=> 'bri_ptmp',
-			echo_can		=> 'mg2',
+			echo_can		=> 'oslec',
 			bri_hardhdlc		=> 'auto',
 			pri_connection_type	=> 'PRI',
 			r2_idle_bits		=> '1101',
